.gitignore

```diff
@@ -12,5 +12,7 @@ node_modules
 certs/
 yarn-error.log
 backend/locust/__pycache__/
+wiremock*.jar
+cypress/cypress.env.json
 frontend/cypress.env.json
```

Comment:
do we still need `frontend/cypress.env.json`?

Reply:
Nope! I'll push an update.

Reply:
Done.